src/styles/blocks/header.scss

```diff
@@ -38,5 +40,11 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    
+    transition: $transition-setting;
```

Comment:
This is not working, as transition only is set to work for `transform` property. You can change it and set to `all` so it works for all the properties that are changing